From 78518a36d7c87bafdbb06478e6b54be5df4b355b Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Mon, 23 Sep 2024 17:49:00 +0100
Subject: [PATCH 1281/1350] dtoverlays: Correct pinctrl assignment on mcp23017

The pinctrl definition used "pinctrl-name" which is missing
an "s" from the end.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/mcp23017-overlay.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm/boot/dts/overlays/mcp23017-overlay.dts
+++ b/arch/arm/boot/dts/overlays/mcp23017-overlay.dts
@@ -34,7 +34,7 @@
 		target = <&mcp23017>;
 		mcp23017_irq: __overlay__ {
 			#interrupt-cells=<2>;
-			pinctrl-name = "default";
+			pinctrl-names = "default";
 			pinctrl-0 = <&mcp23017_pins>;
 			interrupt-parent = <&gpio>;
 			interrupts = <4 2>;
